docs/source/config.rst

```diff
@@ -172,6 +172,10 @@ configuration.
    modules).
    Various model drivers typically define their own default executable names.
 
+``exe_prefix``
```

Comment:
Do we want explicitly say that the `exe_prefix` strings comes before the `exe` but after the `mpirun` (+ any mpi flags)?

Reply:
It's a minor comment but in case that helps future users.